docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -27,7 +27,12 @@
       when: deployment_config is defined
 
     - set_fact:
-        manager_app_chart: "{{ manager_app_chart | default('/usr/local/share/applications/helm/deployment-manager-*.tgz') }}"
+        manager_app_chart: "{{ manager_app_chart }}"
```

Comment:
Those two set_fact tasks are useless unless you want to set default values

Reply:
Have Set the default values